components/xsd-fu/templates-cpp/AggregateMetadata.template

```diff
@@ -397,27 +399,29 @@ namespace ome
         // -- Entity counting (manual definitions) --
 
 {% end debug %}\
+{% if len(model.opts.lang.getSubstitutionTypes()) == 0 %}\
 {% if fu.SOURCE_TYPE == "header" %}\
-        // Documented in base class.
-        index_type
-        getPixelsBinDataCount(index_type imageIndex) const;
+        // Documented in base class.    
```

Comment:
There's a lot of trailing whitespace to clean up here.  Please do a `git grep '[[:space:]]$'` over your changes.